src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,74 @@
 package core.basesyntax;
 
+import static java.lang.Integer.parseInt;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private int countLinesInFile = 0;
 
     public void getStatistic(String fromFileName, String toFileName) {
 
+        String baseString = readToString(fromFileName);
+        String[] sumFields = sumTogether(baseString);
+        saveToFile(sumFields, toFileName);
+    }
+
+    public String readToString(String fileName) {
```

Comment:
lets make all methods except `getStatistic` private, as helper ones(we wont use them outside this class)